python/tests/websockets/test_websocket_security.py

```
@@ -153,6 +153,62 @@ async def echo_handler(websocket: WebSocket):
+    so the oversized value reached the handler. Now it is a hard error.
+    """
+    api = BoltAPI()
+
+    @api.websocket("/ws/echo")
+    async def echo_handler(websocket: WebSocket):
+        await websocket.accept()
+        await websocket.send_text("connected")
+
+    oversized = "a" * 9000  # exceeds the default 8192-byte limit
+
+    with pytest.raises(ValueError) as exc_info:
```

Ruff (0.15.18)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)